abnf/odata-aggregation-testcases.yaml

```diff
@@ -597,35 +598,40 @@ TestCases:
     Rule: odataRelativeUri
     Input: SalesOrganizations?$filter=$it/Aggregation.isleaf(Hierarchy='SalesOrgHierarchy')
 
+  - Name: hierarchy transformations - ancestors with forbidden node property path
+    Rule: queryOptions
+    FailAt: 65
+    Input: $apply=ancestors($root/SalesOrganizations,SalesOrgHierarchy,Sales(4711)/ID,true)
+
   - Name: hierarchy transformations - ancestors
     Rule: queryOptions
     Input: $apply=ancestors($root/SalesOrganizations,SalesOrgHierarchy,ID,contains(Name,'East') or
       contains(Name,'Central'))
 
-  - Name: hierarchy transformations - ancestors
+  - Name: hierarchy transformations - ancestors max distance
     Rule: queryOptions
     Input: $apply=ancestors($root/SalesOrganizations,SalesOrgHierarchy,ID,contains(Name,'East') or
       contains(Name,'Central'), 2)
 
-  - Name: hierarchy transformations - ancestors
+  - Name: hierarchy transformations - ancestors hierarchy directory
     Rule: queryOptions
     Input: $apply=ancestors($root/SalesOrgHierarchies('DEFAULT')/Nodes,SalesOrgHierarchy,ID,contains(Name,'East') or
       contains(Name,'Central'), keep start)
 
-  - Name: hierarchy transformations - ancestors
+  - Name: hierarchy transformations - ancestors node values via navigation property 
```

Comment:
Description would be clearer if input set is mentioned - suggest to rephrase: ancestors processing non-hierarchical input set Sales; node values via navigation property